nixos/modules/services/web-apps/wordpress.nix

```diff
@@ -486,7 +486,7 @@ in
     services.caddy = {
       enable = true;
       virtualHosts = mapAttrs' (hostName: cfg: (
-        nameValuePair "http://${hostName}" {
+        nameValuePair "${hostName}" {
           extraConfig = ''
             root    * /${pkg hostName cfg}/share/wordpress
```

Comment:
While we are here there are a couple of extra slashes in the generated config file:
`root * /${pkg hostName cfg}/share/wordpress` should be `root * ${pkg hostName cfg}/share/wordpress`
and
`php_fastcgi unix/${config.services.phpfpm.pools."wordpress-${hostName}".socket}` should be `php_fastcgi unix${config.services.phpfpm.pools."wordpress-${hostName}".socket}`